mkdocs.yml

```diff
@@ -17,6 +17,7 @@ nav:
   - Python SDK: python.md
   - Private agents: private.md
   - Best practices: best-practices.md
+  - Data Science and Machine Learning on Forta: ds_and_ml.md
```

Comment:
can we name the file with dashes? itll make it more consistent with the other filenames (which become the URLs)